src/c++/perf_analyzer/request_record.h

```diff
@@ -81,6 +86,8 @@ struct RequestRecord {
   std::vector<std::chrono::time_point<std::chrono::system_clock>>
       response_timestamps_;
 
+  // Collection of request inputs
```

Comment:
let's not do comments that add no information. Please remove the previous one on line 91 as well

Reply:
Done